app/store.js

```diff
@@ -0,0 +1,16 @@
+import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
Thanks for encapsulating store creation away from the rest of redux/react. I'd like us to make some of the decisions about this all together. 👍 Thunk or Saga, for example?

Reply:
Agree! =)

Reply:
Suggest moving the store file into the redux folder.

Reply:
@webtechalex Done! =)